packages/modules/src/modules/banners/choiceCardsButtonsBanner/components/ChoiceCardsBannerArticleCount.tsx

```diff
@@ -30,7 +30,7 @@ export function ChoiceCardsBannerArticleCount({
             <ArticleCountOptOutPopup
                 numArticles={numArticles}
                 nextWord=" articles"
-                type="investigations-moment-banner"
+                type="global-new-year-moment-banner"
```

Comment:
what's the effect of changing these values to `global-new-year-moment-banner`?

Reply:
Affect's unused ChoiceCardsBanner, (old) Environment Banners plus WorldPress freedom day article count only. The later had no visual impact I could find hence seemed appropriate to remove/simplify.